### PR TITLE
Add source formatting check

### PR DESCRIPTION
**Pull request by skitaoka**

### CODE REVIEW

File: ci-scripts/osx/travis-build.sh [hunk 1]
```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
+pushd toonz/sources
+chmod +x beautification.sh
```
Comment (janisozaur) [inline ci-scripts/osx/travis-build.sh]: Why not mark it as executable in the repository?

Comment (skitaoka) [inline ci-scripts/osx/travis-build.sh]: Because `./beautification.sh` fail if it did not have the `x`.

Comment (janisozaur) [inline ci-scripts/osx/travis-build.sh]: I think you misunderstood. There's no need to do `chmod +x` after the checkout, you can simply mark it executable in the repository, as it is a script anyway.

Comment (janisozaur) [inline ci-scripts/osx/travis-build.sh]: See https://github.com/opentoonz/opentoonz/pull/694

Comment (skitaoka) [inline ci-scripts/osx/travis-build.sh]: thx. I fixed it on windows by `git update-index --add --chmod=+x toonz/sources/beautification.sh`.

**Comment by shun-iwasawa** [comment]
Jenkins

**Comment by shun-iwasawa** [comment]
LGTM. @skitaoka and @janisozaur thank you for this enhancement!

**Pull request merged**